util.py

```diff
@@ -557,7 +599,7 @@ def set_dxvk_option(opt, val, cfile='/tmp/protonfixes_dxvk.conf'):
     conf = configparser.ConfigParser()
     conf.optionxform = str
     section = conf.default_section
-    dxvk_conf = os.path.join(get_game_install_path(), 'dxvk.conf')
+    dxvk_conf = os.path.join(os.environ['PWD'], 'dxvk.conf')
```

Comment:
I was not sure about this behavior so I replaced this with PWD env var like it used to be before get_game_install_path changes